easybuild/easyconfigs/l/LIANA+/LIANA+-1.0.1-foss-2022a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'LIANA+'
+version = '1.0.1'
+
+homepage = 'https://liana-py.readthedocs.io'
+description = """LIANA+ is a scalable framework that integrates and extends existing methods and knowledge to study
+cell-cell communication in single-cell, spatially-resolved, and multi-modal omics data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('statsmodels', '0.13.1'),
+    ('scanpy', '1.9.1'),
+    ('numba', '0.56.4'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('mudata', '0.2.3', {
+        'checksums': ['45288ac150bfc598d68acb7c2c1c43c38c5c39522107e04f7efbf3360c7f2035'],
+    }),
+    ('mizani', '0.9.3', {
+        'checksums': ['fb61339e9e4711850e902ca286b1ae75255f483823d891aa0515b426d56c606d'],
+    }),
+    ('plotnine', '0.10.1', {
+        'preinstallopts': "sed -i 's/statsmodels>=0.13.2/statsmodels>=0.13.1/g' setup.cfg && ",
```

Comment:
I guess we don't have all the dependencies in place yet to bump LIANA+ itself to a newer toolchain, but is this better than adding a `statsmodels-0.13.2-foss-2022a` easyconfig and an exception to the single dependency version per toolchain test? it doesn't look like a very minor upgrade (https://github.com/statsmodels/statsmodels/compare/v0.13.1...v0.13.2 ...)

Reply:
or simply add `statsmodels` 0.13.2 in this extension list before plotnine?

Reply:
@boegel thoughts on the suggestions from @migueldiascosta ?